sqlitecluster/SQLite.cpp

```diff
@@ -675,6 +652,7 @@ bool SQLite::prepare() {
     // We lock this here, so that we can guarantee the order in which commits show up in the database.
     if (!_mutexLocked) {
         _sharedData.commitLock.lock();
+        _sharedData._commitLockTimer.start("SHARED");
```

Comment:
not sure what SHARED is in this context, the SHARED transactions are those that don't lock the mutex before commit but this is timing the commitLock which is always an exclusive lock.

Reply:
Yes, we're differentiating based on why the lock was locked. Was it locked for the `COMMIT` phase of a parallel (`SHARED`) transaction, or for the entire transaction for a blocking (`EXCLUSIVE`) transaction.

If we find that the mutex is locked 95% of the time, then we're approaching saturation and there's not much we can do about that, unless 50% of the time it's locked for `EXCLUSIVE` trasnactions (quorum, blockingCommit, etc), in which case, those are a big problem. This is a descriptor of why the mutex is locked, not the mode in which it is locked.

Reply:
Okay that makes sense, thanks for the clarification